### PR TITLE
Memberships: Rename to "Recurring Payments"

### PR DESCRIPTION
**Pull request by artpi**
Based on feedback from multiple sources, we've decided to change the name from Memberships  to "Recurring Payments". 

This PR accomplishes that in regards to the block.

## Screenshots

Zrzut ekranu 2019-05-29 o 16 21 02

Zrzut ekranu 2019-05-29 o 16 23 03

## Changes

No functional changes, except naming.

## Testing instructions

Same as https://github.com/Automattic/jetpack/pull/9802

### CODE REVIEW

**Review by davemart-in — DISMISSED** [review]
Looks good to me! 👍

File: extensions/blocks/recurring-payments/edit.jsx [hunk 1]
```diff
@@ -358,11 +358,14 @@ class MembershipsButtonEdit extends Component {
...
 					<Placeholder
 						icon={ <BlockIcon icon={ icon } /> }
-						label={ __( 'Memberships', 'jetpack' ) }
+						label={ __( 'Recurring Payments', 'jetpack' ) }
 						notices={ notices }
 					>
 						<div className="components-placeholder__instructions wp-block-jetpack-membership-button">
-							{ __( "You'll need to upgrade your plan to use the Membership Button.", 'jetpack' ) }
+							{ __(
+								"You'll need to upgrade your plan to use the Recurring Payment Button.",
```
Comment (jeherve) [inline extensions/blocks/recurring-payments/edit.jsx]: Should it be payment**s**?

```suggestion
								"You'll need to upgrade your plan to use the Recurring Payments Button.",
```

Same for the other instances below.

File: extensions/index.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
 		"mailchimp",
 		"map",
 		"markdown",
-		"membership-button",
+		"recurring-payments",
```
Comment (jeherve) [inline extensions/index.json]: You'd need to reorder the list to keep it alphabetically organized I think.

**Review by jeherve — CHANGES_REQUESTED** [review]
This seems to work. Is the Calypso page change coming in wp-calypso as well, I suppose?

I'm not super clear on the naming, and what actually sounds better / is correct English:

- `Recurring Payment plans` or `Recurring Payments plans`
- `Recurring Payments plan` or `Recurring Payments plan`

Also, should we capitalize everything, only Recurring Plans, or only Recurring?

Could we get Editorial's review on this maybe?

**Comment by kristastevens** [comment]
Hi @artpi, thanks for the ping! "Recurring Payments" is clear as is.  

This is likely outside the scope of this PR, though in attempting to view this as a user making a decision on which to choose when offered both options side-by-side,  "Simple Payments" gives me pause -- it's not immediately clear what "Simple" refers to.  (Is it payments processed by Simple? Does Simple mean quick and easy to implement?) Is there any chance we can add micro-copy or an alt-description / tooltip to the block? 

If so, we could have: 

* Recurring Payments Button (Accept recurring payments.)
* Simple Payments Button (Accept one-time payments.)

Thanks again for the ping!

**Comment by artpi** [comment]
Chatted with @kristastevens and we are gonna:
- treat "Recurring Payments" as a "name", so always capitalize
- We are NEVER using "Recurring Payment" since that is not the name. The name has s.
- "Recurring Payments plan" is a thing, as is "Recurring Payments plans". THere are no other forms of plans.

I will implement this in a bit

File: extensions/index.json (hunk 2 above)
Comment (artpi) [reply extensions/index.json]: Changed!

**Comment by artpi** [comment]
I updated the copy according to the above

**Review by jeherve — APPROVED** [review]
This works for me. Merge when ready. I'll make sure the changes make it into Jetpack 7.4.

All those changes mean that you'll need to push those changes to WordPress.com manually though. Fusion won't pick up on all the file name changes. :(

Noting that I'm still redirected to https://en.support.wordpress.com/memberships/#stripe-account-connected upon connecting a site to Stripe.

**Comment by artpi** [comment]
Thank you so much! 🦄

**Comment by jeherve** [comment]
Cherry-picked to `branch-7.4` in fe4fb9a81ff065ec7357ec751c11cf7bd69ebef1

**Pull request merged**